Makefile-test.mk

```diff
@@ -109,6 +112,14 @@ test-multikueue-integration: gomod-download envtest ginkgo dep-crds kueuectl gin
 	$(GINKGO) $(INTEGRATION_FILTERS) $(GINKGO_ARGS) $(GOFLAGS) -procs=$(INTEGRATION_NPROCS_MULTIKUEUE) --race --junit-report=multikueue-junit.xml --json-report=multikueue-integration.json --output-dir=$(ARTIFACTS) -v $(INTEGRATION_TARGET_MULTIKUEUE)
 	$(PROJECT_DIR)/bin/ginkgo-top -i $(ARTIFACTS)/multikueue-integration.json > $(ARTIFACTS)/multikueue-integration-top.yaml
 
+.PHONY: test-multikueue-integration-required
+test-multikueue-integration-required: INTEGRATION_FILTERS= --label-filter="!slow && !redundant"
+test-multikueue-integration-required: test-multikueue-integration ## Run required integration tests for multikueue suites.
+
+.PHONY: test-multikueue-integration-slow-and-redundant
+test-multikueue-integration-slow-and-redundant: INTEGRATION_FILTERS= --label-filter="slow || redundant"
+test-multikueue-integration-slow-and-redundant: test-multikueue-integration ## Run slow and redundant integration tests for multikueue suite.
+
```

Comment:
Actually, for the multikueue tests, there's no need to split them – they pass in 6m24s. https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/kubernetes-sigs_kueue/5068/pull-kueue-test-integration-multikueue-main/1914945114392760320